web/pages/index.md

```diff
@@ -37,7 +37,7 @@ call/cc.
 
 - [Discord Server](https://discord.com/invite/CurfmXNtbN): Most direct way to get support.
 - [Matrix Room](https://matrix.to/#/#k:matrix.org): Another way to get support.
-- [Stackoverflow](https://stackoverflow.com/questions/tagged/kframework): for general questions to the K user community.
+- [Telegram](https://t.me/rv_inc): for newsletters and general announcements.
```

Comment:
I think we should keep StackOverflow, and just add Telegram to thi list perhaps?

Or are we not monitoring stack overflow anymore?

Reply:
FWIW we haven't had a stack overflow question in months; I'm not opposed in principle to keeping the link but we should make sure that we're monitoring it appropriately if we do.

Reply:
The last question we had is from Mar 27, 2022. More than a year ago.

Reply:
huh you're right; I read March and parsed it as this year!